tests/integration/audit_config_base_suite.go

```
@@ -408,3 +448,18 @@ func (s *AuditConfigBaseSuite) checkForPodInStatus(podName string) error {
 
 	return err
 }
+
+// checkForReconciledOperatorVersion Check whether the MondooAuditConfig Status contains the current operator Version after Reconcile.
+func (s *AuditConfigBaseSuite) checkForReconciledOperatorVersion() error {
```

you could move this function somewhere in the test framework instead of defining it in the test suite. In this way it can be re-used by other suites in the future. If you choose not to do it, then you can make it `void` as it doesn't need to return the error. You can already assert in this function

---

I moved all the functions similar to this one into the framework.